### PR TITLE
[Cockroach] Recognize more shapes for now() default from DDL

### PR DESCRIPTION
**Pull request by do4gr**
Fixes https://github.com/prisma/prisma/issues/10665

### CODE REVIEW

**Review by tomhoule — APPROVED** [review]
nice!

File: libs/sql-schema-describer/src/postgres.rs [hunk 1]
```diff
@@ -799,9 +799,13 @@ impl<'a> SqlSchemaDescriber<'a> {
                         ColumnTypeFamily::DateTime => {
                             match default_string.to_lowercase().as_str() {
                                 "now()"
+                                | "now():::timestamp"
+                                | "now():::timestamptz"
+                                | "now():::date"
                                 | "current_timestamp"
                                 | "current_timestamp():::timestamp"
-                                | "current_timestamp():::timestamptz" => DefaultValue::now(),
```
Comment (janpio) [inline libs/sql-schema-describer/src/postgres.rs:804]: Removal of this one is intentional?

Comment (do4gr) [reply libs/sql-schema-describer/src/postgres.rs:804]: It's hiding from you in the line below

Comment (janpio) [reply libs/sql-schema-describer/src/postgres.rs:804]: Indeed. Pattern matching did not anticipate it could change...

**Pull request merged**